### PR TITLE
add command to clear cache and indexdb

### PR DESCRIPTION
**Pull request by linusha**
I am working on implementing #35.

The functionality should be given with the code that already exists (at least works on my maschine^^).
I will add:

- [x] implementing a modal that tells the user that hes about to delete stuff and that the world should be saved beforehand
- [x] add an entry to the world menu to execute the command directly

### CODE REVIEW

**Comment by merryman** [comment]
Nice! And dont bother about the travis build, I have not yet gotten around fixing it.

**Comment by linusha** [comment]
This works now as I would except. Feel free to make suggestions though, I will happily add stuff/make this more robust.

File: lively.ide/world-commands.js [hunk 1]
```diff
@@ -11,9 +11,24 @@ import { LoadingIndicator } from "lively.components";
...
 import { interactivelyFreezeWorld } from "lively.freezer";
 import { resource } from "lively.resources";
+import { BrowserModuleTranslationCache } from "lively.modules/src/instrumentation.js";
 
 var commands = [
 
+  {
+    name: "clear storage and reload",
+    exec: async function() {
+      var proceed = await $world.confirm("Proceeding will clear all local storage and reload the page! Make sure if you want to save your world before proceeding.");
```
Comment (merryman) [inline lively.ide/world-commands.js:21]: I would suggest this formatting of the prompt: 
```
$world.confirm(["Caution\n", {}, "Proceeding will clear all local storage and reload the page! Make sure if you want to save your world before you continue.", { fontSize: 15, fontWeight: 'normal'}], { width: 400 });
```
Improves readability :-)

Comment (linusha) [reply lively.ide/world-commands.js]: Agreed, that is nice!

**Pull request merged**